docs/UserGuide.md

```diff
@@ -156,14 +274,14 @@ Examples:
 
 `personfind Nicole Hardware Display` finds all employees named ‘Nicole’, tagged with BOTH 'Hardware' AND 'Display'
 
-### How to list all persons? `personlist`
+#### How to list all persons? `personlist`
 Lists all the employees in the company.
 
 Format: `personlist`
 
 Example: `personlist` shows all the employees in the company
 
-### How to export a person's information? `export`
+#### How to export a person's information? `export`
```

Comment:
May I know the rationale behind using 4 hashes? We could use single hash titles more often

Reply:
Single hash is same as title format